### DISCUSSION TITLE
Difference between "particle_trajopt.yml" and "gradient_trajopt.yml"

### DISCUSSION THREAD

**Discussion by Cfather** [NVlabs/curobo#233] [Software Q&A]
Hi, I noticed that there are two similar configuration files "particle_trajopt.yml" and "gradient_trajopt.yml". 
Can the values on the same field be different between these two files?

**Comment by balakumar-s** [comment]
particle_trajopt.yml is used by MPPI while gradient_trajopt.yml is used by LBFGS. 

The values are independent so they can be different. We have added documentation for Trajectory Optimization that you might find helpful: https://curobo.org/_api/curobo.wrap.reacher.trajopt.html#module-curobo.wrap.reacher.trajopt
